README.md

```diff
@@ -101,8 +101,9 @@ __fastify-gql__ supports the following options:
 * `loaders`: Object. See [defineLoaders](#defineLoaders) for more
   details.
 * `graphiql`: boolean. Serve
-  [GraphiQL](https://www.npmjs.com/package/graphiql) on `/graphiql` if
-  `routes` is `true`.
+  [GraphiQL](https://www.npmjs.com/package/graphiql) on `/graphiql` and 
+  [GraphQL IDE](https://www.npmjs.com/package/graphql-playground-react) on `/playground`
+  if `routes` is `true`.
```

Comment:
I think we should better expose `playground` with a `playground` option. WDYT?

Reply:
This alongside as graphiql doing pretty much the same thing - providing an simple UI for current graphql server. So I don't see cases where playground could be enabled and graphiql - disabled and vice versa. So the main goal for server developers is that this flag provides UI. And other, for example, FE devs just will choose whatever they like more.

Also if I wrote such plugin just for myself, then I'd have slightly different structure with option called `ui` that will be a prefix for both solutions, e.g. 
```
{
 ui: '/graphql-ui
}
```
And then we have two options that are enabled with such prefix: `/graphql-ui/graphiql` and `graphql-ui/playground`. But `/graphiql` route is kind of de-facto standard for graphql servers and many devs used to it, so this approach would be a bit confusing for people.

One more option - is to have kind of switch between ui implementations: 
```
{
  graphiql: 'graphiql' // one of [undefined, 'graphiql', 'playground']
}
```

Reply:
I think we should support the option as you said, following this pattern:

* if true or ‘graphiql’, enable graphiql
* if ‘playground’, enable the playground

Does that sound reasonable?

Reply:
Yes, sounds good.

Reply:
Done. If you prefer to serve graphql playground on `/graphiql` route when `graphiql: 'playground'` is set, let me know and I'll update the code.